assets/src/edit-story/components/helpCenter/menu/index.js

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+/**
+ * Internal dependencies
+ */
+import { Footer } from './footer';
+import { Header } from './header';
+import { Tips } from './tips';
+import { Transitioner } from './transitioner';
+
+export function Menu({ onTipSelect = () => {}, ...transitionProps }) {
```

Comment:
do we have a noop function util in this codebase? If not, it'd be dope to add one so that it's referentially equal every render